installer/templates/phx_assets/webpack.config.js

```diff
@@ -1,21 +1,14 @@
 const path = require('path');
 const glob = require('glob');
-const HardSourceWebpackPlugin = require('hard-source-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
This is still used but no longer required in package.json 🤔

Reply:
This can go away, it's included in webpack 5: https://webpack.js.org/configuration/other-options/#cache

Reply:
Thanks, TIL!

Reply:
I think the `terser-webpack-plugin` is a transitive dependency. It's pulled in via the `webpack` dependency. But if we use this directly, we should depend on this ourself. 

My editors warns about this:
<img width="1240" alt="Screenshot 2020-11-11 at 17 46 14@2x" src="https://user-images.githubusercontent.com/104180/98839439-d03e0180-2445-11eb-8aed-8ca27695f573.png">

Reply:
Remove of `terser-webpack-plugin`:
I had the same thing in doubt. But as `npm` is strange anyways I removed it.
Will undo it.